xml/admin_rbd.xml

```diff
@@ -979,31 +1114,85 @@ data blocks changed from 2097152 to 2560000</screen>
       to enable it prior to enabling the <option>journaling</option> feature.
      </para>
     </note>
-    <warning>
-     <title>Journaling on All New Images</title>
+    <tip>
+      <para>
+        You can enable journaling on all new images by default by adding
+        <literal>rbd default features = 125</literal> to your Ceph configuration file.
```

Comment:
I like our old version more, where we specify feature names ("layering,exclusive-lock,object-map,deep-flatten,journaling") instead of a cryptic value (125).

Reply:
Note for self:
`rbd default features = layering,exclusive-lock,object-map,deep-flatten,journaling`